mteb/models/overview.py

```diff
@@ -212,22 +212,39 @@ def model_meta_from_hf_hub(model_name: str) -> ModelMeta:
             frameworks.append("Sentence Transformers")
         return ModelMeta(
             name=model_name,
-            revision=None,
+            revision=card.base_model_revision,
```

Comment:
nit: Might want to be consistent with the rest of the file:
```suggestion
            revision=card_data.get("base_model_revision", None),
```